cmd/csi-provisioner/csi-provisioner.go

```diff
@@ -74,6 +74,9 @@ var (
 
 	defaultFSType = flag.String("default-fstype", "", "The default filesystem type of the volume to provision when fstype is unspecified in the StorageClass. If the default is not set and fstype is unset in the StorageClass, then no fstype will be set")
 
+	kubeAPIQPS   = flag.Float32("kube-api-qps", 50, "QPS to use while talking with the kubernetes apiserver. Defaults to 50.0.")
+	kubeAPIBurst = flag.Int("kube-api-burst", 50, "Burst to use while talking with the kubernetes apiserver. Defaults to 50.")
+
```

Comment:
We probably want to have different sets of QPS/Burst config parameters for the leader election client and the client for create/delete operations.

Reply:
I thought about that too, but it seemed a little superfluous to add two more flags just for a leader election client. There's also a snapshot client that uses the same `config`, so that would need its own QPS and Burst too then..

Reply:
I think it's fine for the snapshot client to use the same config as the client that handles create/delete operations.
Let's see if @msau42 has any suggestions here.